### PR TITLE
Add show CSS class to icon for contact list sorting

### PR DESCRIPTION
**Pull request by norkans7**
Fix contact list sorting order indicator icon

### CODE REVIEW

**Comment by norkans7** [comment]
Monosnap Active Contacts 2023-07-27 18-53-48

**Comment by norkans7** [comment]
Monosnap screencast 2023-07-27 19-56-11

File: templates/contacts/contact_list.html [hunk 1]
```diff
@@ -55,6 +55,10 @@
       opacity: 0;
     }
 
+    th:hover a div temba-icon.sorting.hover {
```
Comment (ericnewcomer) [inline templates/contacts/contact_list.html:58]: Not sure why we need the class hover here.

Comment (norkans7) [reply templates/contacts/contact_list.html:58]: Correct, that is not needed I removed that in https://github.com/nyaruka/rapidpro/pull/4764

**Pull request merged**